apps/code-examples/src/app/code-examples/lookup/lookup/async/lookup-async-demo.component.ts

```diff
@@ -1,11 +1,12 @@
 import { Component, OnInit } from '@angular/core';
 import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
 import {
+  SkyAutocompleteSearchAsyncArgs,
   SkyAutocompleteSearchAsyncResult,
   SkyAutocompleteSearchFunctionFilter,
 } from '@skyux/lookup';
 
-import { SkyAutocompleteSearchAsyncArgs } from 'lookup';
+// import { SkyAutocompleteSearchAsyncArgs } from 'lookup';
```

Comment:
We should just remove this commented out line.

Reply:
And remove the import.

Reply:
fixed in commit: [remove commented out import](https://github.com/blackbaud/skyux/pull/52/commits/76f0e7f7507642fe82d60734d564efcd3735b700)